client/src/components/hooks/useSquidList.jsx

```diff
@@ -2,5 +2,10 @@ import { useQuery } from "react-query"
 
 import { ApiClient } from "../../backend/ApiClient"
 
-export const useSquidList = () =>
-  useQuery("squids", () => ApiClient.get("/squids").then((res) => res.data.squids))
+export const useSquidList = (pageOffset, pageSize) =>
+  useQuery(
+    ["squids", { pageOffset }],
```

Comment:
we probably want to keep the page size in the cache key too. Otherwise, you'd get weird results if someone changes the page size.